WcaOnRails/app/views/competitions/_index_table.html.erb

```diff
@@ -1,7 +1,7 @@
 <ul class="list-group">
   <li class="list-group-item"><strong><%= title %></strong></li>
   <% competitions.each_with_index do |competition, index| %>
-    <% if index > 0 && competition.year != competitions[index - 1].year %>
+    <% if index > 0 && competition.year != competitions[index - 1].year && params[:event_ids].empty? %>
```

Comment:
Not saying we should add this right now, but would it be symmetric to add `.break`s that say something like "3 matching events", "2 matching events", "1 matching event"? If you like that idea, there's no need to do it as part of this PR, you can make an issue for it and we can move ahead with merging this up.